packages/diagrams/backend/sirius-components-diagrams-graphql/src/main/java/org/eclipse/sirius/components/diagrams/graphql/datafetchers/subscription/DiagramRefreshedEventPayloadDiagramDataFetcher.java

```diff
@@ -31,8 +31,8 @@ public class DiagramRefreshedEventPayloadDiagramDataFetcher implements IDataFetc
     public DataFetcherResult<Diagram> get(DataFetchingEnvironment environment) throws Exception {
         DiagramRefreshedEventPayload payload = environment.getSource();
         return DataFetcherResult.<Diagram>newResult()
-                .data(payload.getDiagram())
-                .localContext(environment.getLocalContext())
-                .build();
+                                .data(payload.diagram())
+                                .localContext(environment.getLocalContext())
+                                .build();
```

Comment:
Did you change the formatting yourself or is it occurred after a save?

Reply:
It's my formater, it aligns when multiline

Reply:
Could you undo all of your formatting changes?

Reply:
Done